src/parsers/espresso.js

```diff
@@ -1,7 +1,12 @@
 import _ from "underscore";
 import s from "underscore.string";
 
+import { ConstrainedBasis } from "../basis/constrained_basis";
```

Comment:
Ideally, we should have parsers for each version of QE. I don't expect too many variations between versions, but it's important to catch differences if there are some.

I suggest we implement a collection of these parsers where applicable, for instance for version 5.2 and for version 7.1. However, these functions should not be called directly but through an interface that resolves which function should be used (for a given version number). These parsing functions also need to be tested individually, i.e., we need fixtures for each of the versions as well (similar to [Jmol](https://sourceforge.net/p/jmol/code/HEAD/tree/trunk/Jmol-datafiles/quantumEspresso/)).
Since this will undoubtedly grow the lines of code for parsers in this repo, we may also want to move it to a separate package entirely.

These are just my initial thoughts, I'd be happy to discuss this further together.